### PR TITLE
New Map - Synergy

### PR DESCRIPTION
**Pull request by Viciousaurus**
### **Introducing Synergy Station**

synergy9

 (disclaimer: it may not contain synergy).
 (Updated: August 5)

Formerly known as "my hobby during quarantine". Baby's first map so if I fucked up the PR let me know. It has been tested on a local server so it should work.
Synergy it's an average-sized map. It's slightly smaller than box, yet not as small as Packed. It should shine during medpop.

### **Featuring**
Not a whole lot

- Two off-station solar arrays. One of them is only half-complete.
solar

- Engine area that can be used for either the singularity or R-UST. The TEG is facing the area for increased R-UST compatibility.
RUST

- Toxins is on-station, the people that hate the map can erase it from existence.
bomb test

- Small zoo with Inky, Blinky and Pinky. Overly friendly, they should stay inside the habitat at all times.
zoo

- The Vox Trader has a small room in the station to build a custom shop.
voxselling

**And also...**

- Empty space in the middle of the station. It's free real state.

- The kitchen starts with a fish tank, some tools and some eggs. Try something else other than deep fried bread?

- Virology has a window in the cleaning area. Next time you want to scream at the virologist, you can do it without infecting yourself.

- There's no telebay, but Telescience is close and the paramedic has access.

- A basketball court. Can white people jump?

- The Brig's backroom is connected to the courtroom and the IAA's office for a smooth trial.

- Science has some fancy mannequins on display.

- The execution chamber is an incinerator.

**Other zones of interest**

- Mining Outpost
mining

- Reseach Outpost
outpost

- Vox Trader post
vox

- Derelict
derelict

I hope you enjoy downvoting this PR as much as I enjoyed working on it.

<content>

:cl:

 * rscadd: Added Synergy Station

 * rscadd: Added sprites to animal.dmi used in synergy

 * rscadd: Added sprites to mannequin_64x64.dmi, also used in synergy.

 * rscadd: Updated intercom.dm

### CODE REVIEW

**Comment by chesse20** [comment]
looks playable to me, the engine looks like it has a nice medium difficulty singularity to set up; is that a real  or toy singularity in the center of the station though?

**Comment by Viciousaurus** [comment]
> looks playable to me, the engine looks like it has a nice medium difficulty singularity to set up; is that a real or toy singularity in the center of the station though?

Toy

**Comment by cloutcode** [comment]
soul.

**Comment by BurntDevil** [comment]
I havent looked over the rest of the map in high detail, but I did immediately zero in on the bridge. As someone who spends almost every round on the bridge, as bridge officer, I have some minor things to nitpick about your bridge I'd like to see changed.
https://i.imgur.com/GFlgXWG.png
The glass window panels I highlighted might look good, but they keep you from being able to to monitor the CMC and cameras together, as the same for the other side, they impede functionality. These are something I would immediately fireaxe and toss round start. I would suggest removing them, and using tiles to break up the visuals and not use glass to do it.
The SPS monitor I would exchange with the adjacent console. That other one is less used, so sticking it off in the corner with the shuttle controls that are also mostly unused would be a better placement. It would also allow you to man the cmc, look at cameras, and be able to check the SPS without having to run around the side.
Lastly, I would suggest adding intercoms in the departmental sections matching to the ones they serve. Medical intercom by CMC, cargo by cargo consoles, engineering by engineering, security by security. Bridge officers otherwise have to either steal a HoP or Captain headset, or fiddle with a station bouncer. This would make that easier on them.

You also have the accounts database right where it will be immediately stolen, you might want to stuff that somewhere more secure than right by the port side door. Ideally, in the HoP room by the PDA painter.

**Comment by toomykins** [comment]
those solars look like comfy hideouts +1
Couldn't spot much wrong with this in a quick glance-over, maint could do with more lootcrates with random shit in them.

**Comment by Pancakemines** [comment]
Some feedback after a quick look. I don't see a water tank in engineering, should probably be one in atmos for refilling extinguishers. The engineering smes don't seem to do anything at all, as far as I can tell the output wire goes directly into the grilles by them and nowhere else unless its all happening behind that pacman I'm not sure. Maints feel a little bare not even lockers around to hide in or find loot in also I don't see any sunglasses anywhere, gasmasks/voice changers, and a toolbelts are always something worth risking the tunnels for usually. Theres no fueltank/sillicate/water in tool storage which is always the best go to for finding them especially to those without maint access (M'aints) and for the obvious roundstart welderbomb when someone decides they hate the map. I think the toy sing area could do with glass floors by default right now it just seems like a good way to breach the station by accident in any gunfight and having it floored would let someone move in to set up a shop or a bar possibly without that extra risk of a breach. The bridge hallway seems very barren and vision restrictive just a lot of walls maybe some toggle able tinted windows for the main room to see whats going on in that hall. Oxygen and nitrogen having no connection to the mix tank in atmos just feels weird, it stops you using it as a baby burn chamber if you're lazy and prevents you mixing oxygen with n2o for "safe" sleepy floods. The accounts database looks really out of place just tucked between the doors by the bridge entrance and it's also the far side away from the head of personnel who usually maintains it which feels a bit weird. Permabrig having the apc outside with no animal and only one stool prevents the grille on apc escape attempt, I don't know if that's positive or not but it can be quite funny. I also don't see a console to control its injector+scrubbers for putting the prisoners to sleep and clearing it after but I might be blind. That's all I noticed on a quick look.

**Comment by Viciousaurus** [comment]
Alright, now that I fixed the travis issue I'll keep working on the map based on feedback.

**Comment by tuypo1** [comment]
I like it, personally I would have an airlock from the caps office to the bridge but we can see how that goes I think, I am a really big fan of the off station solars

**Comment by Irradiation** [comment]
Capture
You should probably install some shutters in the captain's office, otherwise we're going to have a case of Roid Sparerush 2: Window Breaking boogaloo.

**Comment by west3436** [comment]
~4 of the maint tunnels are pretty big dead ends for most people. I will circle these when I'm at my computer later.

**Comment by StreptoMutans** [comment]
Bhangmeters and Security Alerts Computers should have an intercom right next to it.
Is the white tank in the execution room n2o or air? A port to hook custom canisters would be good, and there should be a direct vent from the atmos into the room to allow air replacing.
Many maps have a radio transmitter for the jukebox, which may be a nice addition.
Why are the buttons for the AI core shutters leading to space and to the Zoo on the outside? Are the shutters designed to keep something in rather than out?
If the shutters at bomb test site only open for a couple seconds after pressing the launch button, they may be closed upon bomb arrival if ZAS rather than the mass driver launches it.
The fancy scrubbers in Toxins storage need a console to be operated.

**Comment by SonixApache** [comment]
We have so many maps already so my mind automatically goes 😕 when I see another one.
Passing over the OP, though, the map looks cute and I look forward to playing on it.

p.s. make away missions

**Comment by ghost** [comment]
Seems like a fun map and engineering looks clean as fuck but where is the maint? The entire maint is just a big two-tile wide hallway. Where is ghetto botany? Where is the maint loot? Where are the extra rooms?

Also that dorms xeno coom room is gonna be a big yikes from me.

**Comment by agentofzog2** [comment]
Maint too tiny. Make sure to put some dumb overpowered shit on the roid or derelict to compensate.

**Comment by SonixApache** [comment]
image

**Comment by Viciousaurus** [comment]
Still working on it, but to answer some questions


> The engineering smes don't seem to do anything at all, as far as I can tell the output wire goes directly into the grilles by them and nowhere else unless its all happening behind that pacman I'm not sure.

It's under the pacman, the output is good.

>I also don't see a console to control its injector+scrubbers for putting the prisoners to sleep and clearing it after but I might be blind. That's all I noticed on a quick look.

Map render bug


> You should probably install some shutters in the captain's office, otherwise we're going to have a case of Roid Sparerush 2: Window Breaking boogaloo.

There's already blast doors in the windows, unless you mean closed by default?

> Edit: the Captain's bedroom isn't connected to his own office because reasons?

Originally the room was inside his office but I thought it wasn't safe enough for the captain to AFK


> Why are the buttons for the AI core shutters leading to space and to the Zoo on the outside? Are the shutters designed to keep something in rather than out?

It's meant to be an emergency entrance/exit to the Bridge, the shutters open a path between the AI and the upload and the button only be activated by head IDs

> If the shutters at bomb test site only open for a couple seconds after pressing the launch button, they may be closed upon bomb arrival if ZAS rather than the mass driver launches it.

Was already tested, its open enough time for the launched object to arrive.

**Comment by Pancakemines** [comment]
It still looks as though the output would be crossing over with the input but assuming that isn't the case the northernmost solars SMES is still superfluous as the wires run directly to the engineering SMES inputs anyway. It'd probably make more sense to have the wires run down to the grid wires in the maint directly south, instead of all the way back to engineering that way you can bypass the SMES on the solars outpost if you wanted to wire it into the grid which is the usual move. 
image

**Comment by Viciousaurus** [comment]
That's fair

**Comment by dunsel-hordon** [comment]
Looks pretty nice. Like others maintenance looks pretty barebones and straight shots. Also no radio transmitter? How dare you! We need music to the whole station. Also some of the hallways should probably have more firelocks.

**Comment by Viciousaurus** [comment]
Updated map. New map can be seen in the OP. Notable changes:

- Changed the Bridge. Windows removed and the computers are placed so it's easier to move from one to another.

- Added maint junk, closets, room.

- Atmos mixing chamber now accepts O2 and N2.

- Added grilles to protect against space debris.

- Solars now wired to the grid.

- Execution chamber now compatible with custom canisters.

**Comment by Pancakemines** [comment]
On behalf of all pipe autists thanks for listening to my pipe/wire issues, final issues would be gas meters on the miner output windows and a meter on the distro pipe after the gas pump in atmos. Those meters are pretty important to see at a glace if the miners are working/pressure checks are off without going into space and the distro one is very important for scripts and glance value to make sure airs in the pipes without scanning anything, it's also nice for obs so they can check distro pressure. I think the usual distro meter tag is dloop_atm_meter but I'm not certain. The mixer into a gas pump for the oxygen and nitrogen is redundant as mixers are inherently gas pumps anyway, I would suggest changing the three way manifolds on the n2/o2 to 4 ways so you have somewhere to hook in as they're connected by default right now you'd be forced to using a socket wrench to hook into those lines which can be quite annoying.

**Comment by bungofungo** [comment]
the bar is nice, are those maint leading into all the service areas usable by service crew?

**Comment by Viciousaurus** [comment]
Updated map. Added to the OP. Notable changes:

- Added pipe meters and other minor atmos changes

- Moved some Brig rooms to the left to make some space.

- Inserted Brig's EVA in that spot. Now is closer to the core of the Brig, instead of behind the Armory.

- Like Roid, the Spare has been moved to the Captain's quarters

**Comment by GreatBigFailure** [comment]
I really like all the available space. It's open but doesn't look too vacant or unfinished. Very conducive to gimmicks or general construction.

**Comment by tuypo1** [comment]
I didn't notice the separate captains bedroom at first, personally I like it

**Comment by gurfan** [comment]
small thing but a table between the wardens office and the rest of the brig would be nice so wardens could distribute stuff over the counter

also I think someone mentioned this already but ghetto bar should have a juke

**Comment by Viciousaurus** [comment]
Updated map, added to the OP. Notable changes:

- Updated [intercom.dm], so instead of only having general and medbay intercoms, now there's engineering, security, cargo and science subtypes. This should come handy for mappers.

- Added Security Intercoms to the Brig and Heads of staff's offices. Now they can call security without using the commn chat or less convenient ways.

- Modified the Bridge again and added doors between rooms, hopefully improving workflow.

- Added table to the Warden's office.

- Added some more firedoors

**Comment by Viciousaurus** [comment]
Quick update

- Added ghetto bar juke

**Comment by Viciousaurus** [comment]
Another quickie, removed things that shouldn't be in space

**Comment by Kammerjunk** [comment]
was expecting to have issues with medbay but it looks extremely well-thought out. main treatment area is sizeable and near chemistry, treatment area and chemistry are close to the entrance, you can access the chem fridge, the sleepers, and the body scanner without blocking traffic, main surgery is visible without being the center of medbay, aux surgery is private, paramed has his own way in and out, morgue is accessible by service staff.
the only thing I can possibly think of is the cloning traffic jam but honestly, I'm not sure that's warranted - complete efficiency isn't fun either

**Comment by Kurfursten** [comment]
Been thinking more about engineering and I don't like that you have to do a big loop to get to and from atmos/engines. There needs to be a good shortcut door. Or alternatively do like Roid and have blast door shortcuts that can be toggled open/closed.

**Comment by ShiftyRail** [comment]
**Brig feedback**

_General comment:_ It's a bit too big, rooms are seemingly out of place, and it lacks doors in some places.

Details:

The processing room looks like it's easily cramped and blocked. It would be better to have so that you can easily circle around the guy being detained, like in Box.

image

Cells 3 and 4 are not visible from outside, which defeats the point of brigging people (what's the use of a guy being put on a timeout if nobody knows about it?)

The front desk has no way to quickly retreat if brig is assaulted, unlike most maps. It's a detail but I believe its design is going to make it painful to use compared to other ones.

The interrogation room proper is a bit too big for something that will rarely be used, especially compared to processing.

EVA airlocks should be in a straight line and the area should not be 1-tile wide.  Traffic jam will be nightmarish with the current proposed design.

I think the pod is out of place and just looks weird, sticking to the wall on one side but 3 tiles away from the other.

That big room with the table is impractical, with no connections to other parts of the brig, oversized, and no windows to space, looking like a dead end. I see no reason for the chargers to be here too. Nobody will actually enter it except for their chargers.

ICly, I don't think you would put the break room in the same spot as the gearing station with the lockers.

HoS office is far too remote to be really integrated. Few people will want to do meetings with their underlings if it takes places out of sight and out of mind.


**In short:** Re-arrange the brig so that the meeting room is less of a dead end, make it easier to navigate, recentre the HoS's office where the officers actually are, separate the locker room to the break room.
The rooms are individually good, they're just not correctly placed.

**Comment by Viciousaurus** [comment]
Alright I think the Conference room in the brig has to go.

> Been thinking more about engineering and I don't like that you have to do a big loop to get to and from atmos/engines. There needs to be a good shortcut door. Or alternatively do like Roid and have blast door shortcuts that can be toggled open/closed.

The shortcut between atmos and engine rooms is on the top left of atmos. Although, now that I think of it, the vents located there will make difficult to put pipes in there. I'll move them somewhere else.

**Comment by Kurfursten** [comment]
Turning on your magboots (slowdown) + walking through 4 airlocks does not make a shortcut. How about this?

image

**Comment by GreatBigFailure** [comment]
The big space hole in the center is gunk waiting to happen. Glass tiles would be a good idea.

**Research outpost**

- There's usually a stack of 30ish plasma somewhere on the outpost. I can't find it in this one. 

- This is minor but a heater and washing machine is normal. The outpost is super convenient for leather making because of it. 

**Medbay**

- There's no exit button for the doors. People without access can't get out.

- With chem and the bar being one tile apart it might be nice to have a way for the barman to pass things to chem for doctor's delight. Might not be an improvement but an idea.

**Bridge**

- The safe spawner on the vault crate might be a bit much. Are those normally there? I'm assuming it's the same kind as the safe so I might be wrong.

- I've never understood why HoP offices sometimes have engineering camera consoles. A cargo computer would make much more sense but again that goes a bit against the norm.

**Comment by Viciousaurus** [comment]
> Turning on your magboots (slowdown) + walking through 4 airlocks does not make a shortcut. How about this?

It's still faster than going the long way, and more than most maps get in terms of atmos-engine accessibility. Still, I'll consider it.



> The big space hole in the center is gunk waiting to happen. Glass tiles would be a good idea.

Embrace the gunk.


> * The safe spawner on the vault crate might be a bit much. Are those normally there? I'm assuming it's the same kind as the safe so I might be wrong.

It has the same contents than the average vault if I'm not mistaken.

> * I've never understood why HoP offices sometimes have engineering camera consoles. A cargo computer would make much more sense but again that goes a bit against the norm.

Those are Mining outpost cameras. Why the icon is a wrench is beyond me.

All the other points are valid and/or oversights on my part.

**Comment by BomberBro** [comment]
What's the point of having a shard on the derelict if there are no radiation collectors, emitters, or plasma tanks? Needs a few of em so an engine can be set up.

**Comment by Viciousaurus** [comment]
Updated the map. Notable changes:

- Restructured the Brig: 

1. Removed the conference room, placed the firing range in its place and the HoS's office is now connected to the Security Office. 

2. Some rooms are smaller.

3. The Prison wing now has 3 cells (down from 4) and a solitary confinement cell. One of the cells has a view to maintenance. Keep it patroled if used.

- Added blast doors connecting engineering and atmos. Pipe away.

- Medbay is no longer a prison

- Added emitters and rad collectors in the derelict

**Comment by Kurfursten** [comment]
Can the MULE actually get past that door to the brig dropoff point?

**Comment by Viciousaurus** [comment]
> Can the MULE actually get past that door to the brig dropoff point?

Thinking about it, I don't think it does. They are fat so they don't move over tiles with windows.
That being said, they are currently broken. Their pathfinding is pretty bad, and instead of moving to a beacon and dropping the crates in the selected area, they drop them over the beacon. To "solve" this I placed the beacon in the delivery area but to make that work I'd have to make the area bigger I suppose, surround the tile with windows instead of placing it over it.
So not sure if I should map them the way they are supposed to work and wait for them to be fixed, or map them the way they currently work like.

**Comment by Viciousaurus** [comment]
After thinking I realized a) the chances of these things getting fixed soon are slim and b) people barely use them, so I fixed the delivery spots to work like they currently do in other maps (crates are left by the flaps). I left a windoor so people won't simply slide under the flaps.

Also added supply time monitors that I had forgotten to add whoops.

**Comment by Kurfursten** [comment]
Reminder that this will not be available for play until pomf adds dmbs
